nautobot_golden_config/__init__.py

```diff
@@ -22,7 +22,6 @@ class GoldenConfig(PluginConfig):
     description = "A plugin for managing Golden Configurations."
     base_url = "golden-config"
     min_version = "1.2.0"
-    max_version = "1.3.99"
```

Comment:
I prefer `max_version = "1.9999"`, if this is the route we're taking.  It not being present at all makes it less clear what we're doing.